ui/src/schemas/index.ts

```diff
@@ -85,9 +95,30 @@ export const arrayOfTelegrafs = [telegraf]
 export const scraper = new schema.Entity(ResourceType.Scrapers)
 export const arrayOfScrapers = [scraper]
 
-export const addLabels = <R extends {labels?: Label[]}>(resource: R): R => {
-  return {
-    ...resource,
-    labels: (resource.labels || []).map(addLabelDefaults),
+/* Variables */
+
+// Defines the schema for the "variables" resource
+export const variable = new schema.Entity(
+  ResourceType.Variables,
+  {},
+  {
+    processStrategy: (v: Variable): Variable => {
+      return {
+        ...v,
+        labels: addLabels(v),
+        status: addStatus(v),
+      }
+    },
   }
+)
+export const arrayOfVariables = [variable]
```

Comment:
what's this all about? doing a `cmd+f` for it doesn't show it used in this pr

Reply:
Should normalizer want to normalize a list of a foo it uses this syntax for it's schema.  I could just inline this, but (imo), `variable` and `[variable]` looked to similar when consumed as arguments to the `normalize` function i.e.:

`normalize(data, fooSchema)` and `normalize(data, [fooSchema])` looked really similar but return different results.  So, to make it easier on the eye I assigned all resource list schemas to a variable.

Reply:
I can remove it if it's not used.